app/operations/assignments/create.rb

```diff
@@ -1,7 +1,7 @@
 module Assignments
   class Create < Operation
-    integer :base_project_id, default: Rails.application.secrets["zooniverse_project_id"]
```

Comment:
Yes. Project ID will be required for all requests and workflow ID is required for projects with static workflows/subject sets.